client/src/components/App.jsx

```diff
@@ -15,7 +15,7 @@ import TrackerContext from '../utilities/TrackerContext.js';
 
 function App() {
   const trackClick = useContext(TrackerContext);
-  const [productID, setProductID] = useState(40352);
+  const [productID, setProductID] = useState(40349);
```

Comment:
- possibly trivial comment, but, we should have team chat about which ID we want to default to for presentation & deployment
- (nothing to change right now)